inductiva/projects/project.py

```diff
@@ -0,0 +1,102 @@
+"""Project class"""
+import contextvars
+
+import inductiva
+from inductiva.client import ApiException
+from inductiva.client.apis.tags import projects_api
+
+CURRENT_PROJECT = contextvars.ContextVar("current_project", default=None)
```

Comment:
Can we make this a class var? like this `Project.CURRENT_PROJECT`

Reply:
I think I prefer to have separate from the class. For example, later on, on submitting a task I think that it is more intuitive to just use something like:

```python
task = api.task_submit(..., project=inductiva.get_current_project())
```